readme.md

```diff
@@ -655,6 +655,15 @@ await got('https://api6.ipify.org', {
 });
 ```
 
+###### lookup
```

Comment:
```suggestion
###### dnsLookup
```

?

Reply:
Actually is called `lookup` should we change the name?

Since we are near Got 12 I think the name should be changed in it.

Reply:
Yeah, I was asking if you should rename it. I think we should.

Reply:
Let's leave it as `lookup` and then change it in Got 12. If there are going to be a reasonable number of dns related options we could create a `dns` option that includes all of them.

Reply:
Added it to https://github.com/sindresorhus/got/issues/1466